python/tank/authentication/user_impl.py

```diff
@@ -206,7 +206,19 @@ def __init__(self, host, login, session_token, http_proxy, password=None, cookie
         self._login = login
         self._session_token = session_token
         self._cookies = cookies
-        self._saml_expiration = saml_expiration
+
+        # saml expiration is meaningful only with SSO enabled.
+        # The presence of cookies implies potential SSO usage.
```

Comment:
for anyone doing support on SSO, would debug logging here help at all? I suggest thinking through the story where tannaz or phil have to debug some weird sso problem and ask for the debug logs: would it be helpful to display some of the cookie contents? would it be useful to log the expiration times set? etc.